packages/builder/src/index.html

```diff
@@ -21,6 +22,9 @@
 </head>
 
 <body id="app">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.7.4/d3.js"></script>
```

Comment:
I think we may want to bundle these with the standard-components rather than include them from a CDN

Reply:
Makes sense but all the charts are bundled as UMD module format and we can't use require syntax in Svelte, right?